### ISSUE TITLE
About training loss

### ISSUE THREAD

**Issue by Zhshao11** [HeatherJiaZG/SuperGlue-pytorch#36]
Why the loss function is calculated using "all_matches"? 
In the code, it seems that "all_matches" is computed using SIFT features, rather than the partial assignment matrix described in the paper.
